c/serverStatusService.c

```diff
@@ -0,0 +1,266 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+
+#ifdef METTLE
+#include <metal/metal.h>
+#include <metal/stddef.h>
+#include <metal/stdio.h>
+#include <metal/stdlib.h>
+#include <metal/string.h>
+#include "metalio.h"
+#include "qsam.h"
+#else
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#endif
+#include <sys/utsname.h>
+
+#include "zowetypes.h"
+#include "bpxnet.h"
+#include "utils.h"
+#include "socketmgmt.h"
+
+#include "httpserver.h"
+#include "json.h"
+#include "logging.h"
+#include "zssLogging.h"
+#include "restService.h"
+#pragma linkage (EXSMFI,OS)
+
+#ifdef __ZOWE_OS_ZOS
+static int serveStatus(HttpService *service, HttpResponse *response);
+static JsonObject* serverConfig = NULL;
+char productVersion[40];
+extern char **environ;
+
+typedef int EXSMFI();
```

Comment:
the correct syntax for a function with no arguments is` EXSMFI(void)`. `EXSMFI()` means "a function which you can all with any number of arguments, all of them will be cast to int"